app/models/patient_import.rb

```diff
@@ -17,6 +20,21 @@ def count_column(patient, parents, parent_relationships)
     end
   end
 
+  def validate_pds_match_rate!
+    return if valid_pds_match_rate? || changesets.count < CHANGESET_THRESHOLD
+
+    update_column(:status, :low_pds_match_rate)
```

Comment:
```suggestion
    update!(status: :low_pds_match_rate)
```

Should we use `update!` to ensure `updated_at` gets set?